core/package.json

```diff
@@ -8,12 +8,12 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "postinstall": "./init.sh",
-    "build": "turbo run build",
-    "test": "turbo run test",
-    "lint": "turbo run lint",
-    "format": "turbo run format",
-    "size": "turbo run size",
-    "coverage": "turbo run coverage",
+    "build": "turbo run build --filter '!./packages/contracts/lib/*'",
+    "test": "turbo run test --filter '!./packages/contracts/lib/*'",
+    "lint": "turbo run lint --filter '!./packages/contracts/lib/*'",
+    "format": "turbo run format --filter '!./packages/contracts/lib/*'",
```

Comment:
Is the `filter` here nessessary?

Reply:
yeah, unfortunately. Otherwise turborepo will discover all npm script commands in the lib dir and run them.

But I am thinking we should remove turborepo since its a bit overkill for the few packages we have.